content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -0,0 +1,238 @@
+---
+title: With Gateways
+description: Istio mesh spanning multiple Kubernetes clusters using Istio Gateway to reach remote pods.
+weight: 2
+keywords: [kubernetes,multicluster,federation,gateway]
+---
+
+Instructions for spanning an Istio mesh across multiple clusters when pods
```

Comment:
IMO - "spanning" is the wrong description here.  The mesh doesn't span the clusters each mesh is independent and separately controlled by a pilot.  If you feel strongly about "spanning" I am not that hung up on the name.

Reply:
Identical meshes.. with configuration being replicated. Independent meshes mean independent administrators, lack of visibility over the internal services in each cluster, lack of control over the configuration/routing rules etc. None of which applies here. Its just two clusters, with two pilots that compute the same configs

Reply:
I would choose to describe it differently, but not a major point and subject to interpretation so I will leave it to your judgement.

Reply:
@rshriram Please describe it differently. The term span is not appropriate, see my previous comments. The sentence has several grammar issues, beginning with it being a sentence fragment. My suggestion would be:
This procedure details how to install an Istio mesh across multiple clusters when the pods within each cluster connect exclusively to remote gateways using IP addresses.